src/browser/components/SkillItem.react.js

```diff
@@ -16,46 +15,23 @@ export default class SkillItem extends Component {
   handleChange(level, e) {
     const {actions: {saveSkill}} = this.props;
     saveSkill(this.props.data, level);
-    this.props.resetErrorStatus();
   }
 
-  setHintVisibility(visible) {
-    this.setState({hintDisplayed: visible});
-  }
-
-  handleMouseEnter = () => this.setHintVisibility(true);
-  handleMouseLeave = () => this.setHintVisibility(false);
-
   render() {
-    let hint;
-    const cb = {
-      onMouseEnter: this.handleMouseEnter,
-      onMouseLeave: this.handleMouseLeave
-    };
-
-    if (this.state.hintDisplayed) {
-      hint = (<div {...cb} className="hint">{this.props.data.name}</div>);
+    const starsLabels = [];
+    for (let i = 5; i >= 1; i--) {
+      starsLabels.push(
+          <input className={`star star-${i}`} id={`star${i}_${this.props.data.id}`} key={i} name="star" onChange={(e) => this.handleChange(i, e)} type="radio"/>,
+          <label className={`star star-${i}`} htmlFor={`star${i}_${this.props.data.id}`} key={i + 5} />
```

Comment:
`this.handleChange` could be more descriptive, since component is not responsible only for showing just the stars. Sth like `this.handleStarChange` should be enough